compiler/damlc/tests/BUILD.bazel

```
@@ -85,6 +85,7 @@ da_haskell_test(
     data = [
         "//compiler/damlc",
         "//daml-script/daml:daml-script.dar",
+        "//daml-script/daml:daml-script-1.dev.dar", # TODO(MA): remove once we are on DAML-LF 1.15
```

Could you add a link to the 1.15 issue here, and at least once in DamlcTest.hs? e.g.

```suggestion
        "//daml-script/daml:daml-script-1.dev.dar", 
        # TODO https://github.com/digital-asset/daml/issues/12051
        #   Remove once DAML-LF 1.15 is the default compiler output
```

---

will do, thanks!